skimage/morphology/tests/test_footprints.py

```diff
@@ -157,7 +157,7 @@ def test_footprint_dtype(function, args, supports_sequence_decomposition, dtype)
 
 
 @pytest.mark.parametrize("function", ["disk", "ball"])
-@pytest.mark.parametrize("radius", [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 50, 75, 100])
+@pytest.mark.parametrize("radius", [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20])
```

Comment:
Is this change unrelated to the rest?

Reply:
Yes, see https://github.com/scikit-image/scikit-image/pull/7665/commits/69c82eda3ee4d6c6fa6f96d621575a6d6f8bbbff:

> Remove slow parameters in test_nsphere_series_approximation 
>
> Creating and testing large 3D footprints with ball is very slow so that
the test took 26 s in total on my machine. Removing 50-100 shaves of
~25 s without sacrificing any test coverage I think.

Reply:
"shaves off" (I guess)

Thanks. I admit I didn't review per commit this time!